src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -4991,11 +5044,11 @@ public partial class FileStream : System.IO.Stream
         public virtual System.IntPtr Handle { get { throw null; } }
         public virtual bool IsAsync { get { throw null; } }
         public override long Length { get { throw null; } }
-        public string Name { get { throw null; } }
+        public virtual string Name { get { throw null; } }
```

Comment:
There is a bunch of unrelated changes like this one. Are they all intentional?

Reply:
Yes these are intentional. They are essentially refreshing based on what is in System.Private.CoreLib implementation. A number of changes to the API never made it to the ref.